index.js

```diff
@@ -18,6 +19,28 @@ const TimerMixin = require('react-timer-mixin');
 const DefaultTabBar = require('./DefaultTabBar');
 const ScrollableTabBar = require('./ScrollableTabBar');
 
+class StaticContainer extends Component{
```

Comment:
i dint know about this. Logic's no different. Let me know, i can change it if you insist